CRM/Case/XMLProcessor.php

```diff
@@ -107,27 +89,41 @@ public function &allActivityTypes($indexName = TRUE, $all = FALSE) {
   }
 
   /**
+   * Get all relationship type labels
+   *
+   * TODO: These should probably be names, but under legacy behavior this has
+   * been labels.
```

Comment:
The "labels instead of names" thing was a design mistake that's haunted us for way too long. Every time a piece of CiviCase gets refactored this problem gets punted as "out of scope" for the issue at hand. I understand that, but jeeze this is a frustrating problem!

Reply:
At least now there's an [explicit ticket for it](https://lab.civicrm.org/dev/core/issues/856), even though I'm [not a fan](https://github.com/civicrm/civicrm-core/pull/14001#issuecomment-496544294) of the solution proposed in #14001.

Reply:
@agh1 @colemanw That PR wasn't intended as a proposed solution for name vs. label - its intention seems to be to just allow changing a relationship label in the UI which happens to be blocked because of name vs label. In https://lab.civicrm.org/dev/core/issues/774#note_14566 I had started on a WIP but put it on hold since it is a bit of work and has a blocker or two.